BedrockServer.cpp

```diff
@@ -1284,32 +1288,39 @@ void BedrockServer::_status(BedrockCommand& command) {
         response.content = SComposeJSONObject(content);
     }
 
-    else if (SIEquals(request.methodLine, STATUS_WHITELIST)) {
-        SAUTOLOCK(_parallelCommandMutex);
+    else if (SIEquals(request.methodLine, STATUS_BLACKLIST)) {
+        SAUTOLOCK(_blacklistedParallelCommandMutex);
 
         // Return the old list. We can check the list by not passing the "Commands" param.
         STable content;
-        content["oldCommandWhitelist"] = SComposeList(_parallelCommands);
+        content["oldCommandBlacklist"] = SComposeList(_blacklistedParallelCommands);
 
         // If the Commands param is set, parse it and update our value.
         if (request.isSet("Commands")) {
-            _parallelCommands.clear();
+            _blacklistedParallelCommands.clear();
             list<string> parallelCommands;
             SParseList(request["Commands"], parallelCommands);
             for (auto& command : parallelCommands) {
-                _parallelCommands.insert(command);
+                _blacklistedParallelCommands.insert(command);
             }
         }
         if (request.isSet("autoBlacklistConflictFraction")) {
             BedrockConflictMetrics::setFraction(SToFloat(request["autoBlacklistConflictFraction"]));
         }
 
         // Enable extra logging in the commit lock timer.
-        decltype(SQLite::g_commitLock)::enableExtraLogging.store(!_parallelCommands.empty());
+        decltype(SQLite::g_commitLock)::enableExtraLogging.store(!_blacklistedParallelCommands.empty());
 
-        // PRepare the command to respond to the caller.
+        // Prepare the command to respond to the caller.
         response.methodLine = "200 OK";
         response.content = SComposeJSONObject(content);
+    } else if (SIEquals(request.methodLine, STATUS_MULTIWRITE)) {
+        if (request.isSet("Enable")) {
```

Comment:
If you set `Enable: false` this will still return `true`, right?  Do you mean to use `request.test("Enable")`?

Reply:
ooh, got it, that's intentional.  nm.